indico/modules/rb_new/client/js/modules/bookRoom/BookRoom.jsx

```diff
@@ -27,14 +28,21 @@ import LazyScroll from 'redux-lazy-scroll';
 import {Slot, toClasses} from 'indico/react/util';
 import {PluralTranslate, Translate, Singular, Param, Plural} from 'indico/react/i18n';
 import mapControllerFactory from '../../containers/MapController';
-import BookingFilterBar from '../BookingFilterBar';
 import filterBarFactory from '../../containers/FilterBar';
 import searchBoxFactory from '../../containers/SearchBar';
 import Room from '../../containers/Room';
-import BookingTimeline from '../BookingTimeline';
-import BookRoomModal from '../../containers/BookRoomModal';
-import roomDetailsModalFactory from '../modals/RoomDetailsModal';
-import {roomPreloader} from '../../util';
+import BookingFilterBar from './BookingFilterBar';
+import BookingTimeline from './BookingTimeline';
+import BookRoomModal from './BookRoomModal';
+import roomDetailsModalFactory from '../../components/modals/RoomDetailsModal';
+import {roomPreloader, pushStateMergeProps} from '../../util';
+import {queryString as qsFilterRules} from '../../serializers/filters';
+import {rules as qsBookRoomRules} from './queryString';
+
+import * as actions from './actions';
```

Comment:
`bookRoomActions`? That way it's easier to search for all places calling a certain action.